plugin/src/Config.lua

```diff
@@ -3,8 +3,9 @@ local strict = require(script.Parent.strict)
 local isDevBuild = script.Parent.Parent:FindFirstChild("ROJO_DEV_BUILD") ~= nil
 
 local Version = script.Parent.Parent.Version
-local realVersion = Version.Value:split(".")
+local major, minor, patch, metadata = Version.Value:match("^(%d+)%.(%d+)%.(%d+)(.+)$")
```

Comment:
This will fail in the case of e.g. `7.4.0` because the ending does not match `(%d+)(.+)`

```suggestion
local major, minor, patch, metadata = Version.Value:match("^(%d+)%.(%d+)%.(%d+)(.*)$")
```